src/ViewHelpers/CarViewHelper.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+
```

Comment:
do you need to remove this space - not sure it it's one of two line gap needed between the php tag and namespace